src/visualizations/store/index.js

```diff
@@ -31,12 +31,14 @@ export default function ({
         type,
         seriesId = initialSeriesId,
         categoryIds = [initialCategoryId],
+        extraOptions,
```

Comment:
Related to above comment, should this default to and empty object?

Reply:
Initialized as empty object.